recipe/conda_build_config.yaml

```diff
@@ -133,54 +147,55 @@ target_c_stdlib_version:
   - 10.13                               # [osx and x86_64]
   # For osx-arm64 -> win-64 cross-compiling, set SDK version for the build system
   - 11.0                                # [osx and not x86_64]
+  - 0                                   # [win64]
 version:
   - 1.86.0  # [linux]
   - 1.86.0  # [linux]
   - 1.86.0  # [linux]
   - 1.86.0  # [linux64 or osx]
   - 1.86.0  # [linux64 or osx]
-  - 1.86.0  # [linux64 or win64]
-  - 1.86.0  # [osx]
-
+  - 1.86.0  # [linux64 or osx or win64]
+  - 1.86.0  # [win64]
+  
   - 1.88.0  # [linux]
   - 1.88.0  # [linux]
   - 1.88.0  # [linux]
   - 1.88.0  # [linux64 or osx]
   - 1.88.0  # [linux64 or osx]
-  - 1.88.0  # [linux64 or win64]
-  - 1.88.0  # [osx]
+  - 1.88.0  # [linux64 or osx or win64]
+  - 1.88.0  # [win64]
```

Comment:
So `# [win64]` covers both `win-64` and `win-arm64`, but `linux64` doesn't cover `linux-aarch64`? That's a recipe for confusion IMO 😑

Reply:
Or perhaps `# [linux64]` literally only means "64bit linux"? If so, then I was fooled by the similarity between `linux-64` and `linux64` for the last few years; I had always thought the latter also implied x86_64 (and it seems to have worked like that in several tricky zips). 🤔

Reply:
Yeah we should steer towards a set of selectors with clearly defined behavior

Wrote up some thoughts in issue: https://github.com/conda-forge/conda-smithy/issues/2305

Reply:
Yeah I have also had a lot of trouble wrapping my head around this..

Reply:
If either of you have thoughts on how things should look, please comment in that conda-smithy issue

Reply:
This is just misinformation. `linux64` means the subdir `linux-64` and `win64` means the subdir `win-64`.  The selector here means the compiler is run in `win-64`, but the compiler targets `win-arm64`.